### PR TITLE
Update README with better description for Charmhub

### PR DESCRIPTION
**Pull request by heitorPB**
Solves INFRA-655

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,47 @@
 # Omnivector Slurm Distribution Bundles
 
-This repository is a companion to
-[Slurm Charms](https://github.com/omnivector-solutions/slurm-charms/).
+A suite of codified operations to assemble, install, deploy, and operate
```
Comment (dusktreader) [inline README.md:3]: This is a sentence fragment. Maybe:
```suggestion
This suite provides a set of codified operations to assemble, install, deploy, and operate
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,47 @@
...
+```bash
+$ juju deploy slurm
+```
+
+This command will deploy and configure a Slurm cluster with:
+- 1 `slurmctld` node: the central management node
+- 1 `slurmdbd` node: the database accounting node
+- 1 `slurmd` node: the compute node
+- 1 `slurmrestd` node: the REST API interface to Slurm
+- 1 `percona-cluster` node: the MySQL database
```
Comment (dusktreader) [inline README.md:24]:
```suggestion
This command will deploy and configure a Slurm cluster with one of each of the following nodes:
- `slurmctld`: the central management node
- `slurmdbd`: the database accounting node
- `slurmd`: the compute node
- `slurmrestd`: the REST API interface to Slurm
- `percona-cluster`: the MySQL database
```

**Review by dusktreader — CHANGES_REQUESTED** [review]
Very nice update. Added a few comments for you.

File: README.md (hunk 2 above)
Comment (heitorPB) [reply README.md]: Oh, way better, thanks!

**Review by dusktreader — APPROVED** [review]

**Pull request merged**